torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -385,6 +389,14 @@ def fsdp_modules(
 
         return fsdp_modules
 
+    def _fsdp_modules_with_params(self) -> List["FullyShardedDataParallel"]:
```

Comment:
I think we have a .fsdp_modules() interface already, could we just add a `with_params` attribute to that instead of a  new function?